tests/UserContainerTests.cpp

```diff
@@ -0,0 +1,68 @@
+#include <gtest/gtest.h>
+
+#include "UserContainer.hpp"
+#include "User.hpp"
+
+struct UserContainerTests : public ::testing::Test
+{
+    UserContainerTests()
+    {
+        userContainer_.createUser(User("Kamil", "Kamil.Waszkiewicz", "wieczorowa", "Kamil.Waszkiewicz", "Kamil.Waszkiewicz"));
+    }
+
+    UserContainer userContainer_;
+};
+
+TEST_F(UserContainerTests, canConstructUserContainer)
+{
+    UserContainer uc;
```

Comment:
test not needed, because other tests always construct UserContainer, so it is covered by them.

Reply:
Fixed